packages/package_gb_taxonomy_8d245994d7/tool_dependencies.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="gb_taxonomy" version="8d245994d7">
+        <install version="1.0">
+            <actions>
+                <action type="shell_command">git clone https://github.com/spond/gb_taxonomy_tools.git gb_taxonomy</action>
+                <action type="shell_command">git reset --hard 8d245994d77546daac1f595b87a110a620426b4c</action>
+                <action type="shell_command">cmake -D INSTALL_PREFIX:PATH=$INSTALL_DIR .</action>
```

Comment:
I seem to remember an IUC cmake dependency making it into the TS. Would you mind depending on that? https://github.com/galaxyproject/tools-iuc/pull/210